qgis_deployment_toolbelt/profiles/rules_context.py

```diff
@@ -14,6 +14,7 @@
 # Standard library
 import json
 import logging
+import os
```

Comment:
Prefer complete function import rather than package. See other projects modules where we work with env vars to make it consistent with.